### ISSUE TITLE
Firefox extension outdated

### ISSUE THREAD

**Issue by michael1900** [0x48piraj/fadblock#234]
Hi all! 

Is it possible to update also the Firefox extension?

Thank you to Developer for the big work!

**Comment by 0x48piraj** [comment]
I have pushed the update on Firefox, it will take some time to get published.

**Comment by n1mrod88** [comment]
The firefox version stopped working for me today.

**Comment by ckeene159** [comment]
Same thing happened to me yesterday with Firefox. I don't know how to tell if I ever had Fadblock v2.0. I did log in to Fadblock after v2.0 was announced, something I hadn't bothered with previously even though I'd donated. I never saw any change, but I had set Firefox to automatically clear cache on closing. So what it was doing, both before and after I logged in to Fadblock, is it would play the ad fast but then freeze on a "skip" option. Yesterday it stopped playing the ads fast, so it's just like no Fadblock. I wish I could tell 1) what version of Fadblock I have, and 2) whether I'm logged in to my Fadblock account. I assume I am logged in as it no longer offers an option to log in.

**Comment by 0x48piraj** [comment]
Yeah, I have updated the extension but unfortunately the Firefox team is delaying the release.

**Comment by 0x48piraj** [comment]
@ckeene159, I am currently looking into the issue, I believe it stopped working for many people since yesterday as probably YT again pushed a countermeasure regionally.
